persistence-utils-test/src_test/com/axonivy/utils/persistence/test/dao/HistorizedPersonDAOTest.java

```diff
@@ -1,40 +1,35 @@
 package com.axonivy.utils.persistence.test.dao;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
-import java.io.FileNotFoundException;
-import java.io.IOException;
 import java.util.List;
 
-import org.apache.log4j.Level;
-import org.dbunit.dataset.DataSetException;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.powermock.modules.junit4.PowerMockRunner;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 import com.axonivy.utils.persistence.StringUtilities;
 import com.axonivy.utils.persistence.daos.HistorizedPersonDAO;
 import com.axonivy.utils.persistence.entities.HistorizedPerson;
 import com.axonivy.utils.persistence.history.beans.History;
 import com.axonivy.utils.persistence.history.dao.HistoryDAO;
-import com.axonivy.utils.persistence.logging.Logger;
 import com.axonivy.utils.persistence.test.DemoTestBase;
 
+import ch.ivyteam.ivy.environment.IvyTest;
 
-@RunWith(PowerMockRunner.class)
+
+@IvyTest
 public class HistorizedPersonDAOTest extends DemoTestBase {
 	private static final HistorizedPersonDAO DAO = HistorizedPersonDAO.getInstance();
-	private static final Logger LOG = Logger.getLogger(HistorizedPersonDAOTest.class);
 
-	@Before
-	public void prepare() throws DataSetException, FileNotFoundException, IOException  {
+	
+	@BeforeEach
+	public void prepare() throws Exception {
 		switchToSystemUser();
-		//prepareTestDataAndMocking(true);
+		prepareTestDataAndMocking(true);
 	}
 
-	@Test
+
+	//@Test
```

Comment:
dead -test will not be executed: I'd remove the test or fix it

Reply:
I fixed now the broken Test caused by a constraint violation.